src/styles/button.scss

```diff
@@ -0,0 +1,19 @@
+.card__link-by {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: $width;
+  width: 166px;
+  font-size: 14px;
+  font-weight: bold;
+  text-transform: uppercase;
+  color: #fff;
+  background-color: $color-link;
+  border-radius: 5px;
+  border: 1px solid $color-link;
```

Comment:
[Checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md) 12 point. Check this in all places